compiler/rustc_infer/src/infer/error_reporting/suggest.rs

```diff
@@ -283,6 +268,20 @@ impl<'tcx> TypeErrCtxt<'_, 'tcx> {
         }
     }
 
+    pub fn suggest_await_on_future(
+        &self,
+        diag: &mut Diagnostic,
+        sp: Span,
+        _cause: &ObligationCause<'tcx>,
```

Comment:
Why does this take a cause if it's not used?

Reply:
just a refused, thanks!